arch/lkl/kernel/threads.c

```diff
@@ -5,7 +5,7 @@
 #include <asm/cpu.h>
 #include <asm/sched.h>
 
-static volatile int threads_counter;
```

Comment:
Strange that no "threads_counter" is defined in my version ( based on a063e1631db5 - Merge pull request #340 from libos-nuse/fix-checkpath-tak2).